NEWSINDEX

```diff
@@ -1,59 +1,59 @@
-20
-58
-59
-62
-65
-67
-68
-69:the-coq-workshop-2009
-70:announcing-lngen
-71:a-locally-nameless-backend-for-ott
-72:first-asian-pacific-coq-summer-school
-78:a-tactic-for-deciding-kleene-algebras
-81:coq-82pl1-is-out
-83:announcing-ssreflect-version-12
-91:coq-83-beta-version
-93:coq-workshop-2010
-94:2nd-asian-pacific-coq-summer-school
-95:alpha-release-of-coq-modulo-theories
-96:coq-83-is-out
-98:coq-83pl2-is-out
-99:3rd-asian-pacific-summer-school-on-formal-methods
-100:coq-workshop-2011
-105:coq-83pl3-is-out
-106:beta-release-of-coq-84
-107:release-candidate-of-coq-84-is-out
-110:coq-84-is-out
-114:coq-received-acm-sigplan-programming-languages-software-2013-award
-115:coq-source-repository-migrated-to-git
-119:coq-received-acm-software-system-2013-award
-120:coq-84pl4-is-out
-121:coq-is-hiring-a-specialized-engineer-for-2-years
-122:coq-84pl5-is-out
-123:coq-85beta1-is-out
-124:coq-84pl6-is-out
-125:coq-85beta2-is-out
-126:coq-85beta3-is-out
-127:coq-85rc1-is-out
-128:coq-85-is-out
-129:coq-85pl1-is-out
-130:coq-85pl2-is-out
-131:coq-85pl3-is-out
-132:coq-86beta1-is-out
-133:coq-86rc1-is-out
-134:coq-86-is-out
-135:coq-861-is-out
-136
-137:coq-87beta1-is-out
-138:coq-87beta2-is-out
-139:coq-870-is-out
-140
-141:coq-871-is-out
+coq-890-is-out
```

Comment:
Isn't this more or less the content of the `news` folder? Maybe we could keep `RECENTNEWS` and remove this list?

Reply:
How do you sort the news then? This file lists them in chronological order.

Reply:
What about prefixing the filename with a number. E.g. `141-coq-871-is-out`, etc.

Reply:
@vbgl Do you mean something as `2019-01-28-blabla` or the actual timestamp of the file? If you meant the latter, then I disagree, because it would prevent fixing typos.

Reply:
I think we were talking about naming scheme of news files.

I think that keeping a list is a good way to define a total order. It should be noted that this no more data duplication than storing the date in the file name as well as part of its contents.

Moreover, I should emphasize that we are here designing a transient situation, as we plan to switch to a more main-stream static website generator.